seekers/tests.py

```diff
@@ -41,32 +33,29 @@ def test_seeker_form_validity(city, start_date, min_rent, max_rent, num_of_rooma
     assert form.is_valid() is validity
 
 
-@pytest.fixture
-def valid_form(new_city):
-    return SeekerCreationForm(data={
-        'city': new_city,
-        'start_date': '1900-1-1',
-        'min_rent': 1000,
-        'max_rent': 4000,
-        'num_of_roomates': 2,
-        'num_of_rooms': 3,
-        'about': 'about',
-    })
-
-
 @pytest.mark.django_db
-def test_valid_form_is_valid(valid_form):
-    assert valid_form.is_valid
+def test_valid_form_is_valid(valid_seeker_creation_form):
+    assert valid_seeker_creation_form.is_valid
 
 
 @pytest.mark.django_db
-def test_fail_to_save_seeker_form_with_commit_true(valid_form):
+def test_fail_to_save_seeker_form_with_commit_true(valid_seeker_creation_form):
     with pytest.raises(ValueError):
-        valid_form.save(commit=True)
+        valid_seeker_creation_form.save(commit=True)
 
 
 @pytest.mark.django_db
 def test_register_apartment_view(client):
     url = reverse('register_seeker')
     response = client.get(url)
     assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_accessing_seeker_register_view_with_logged_user(client, user_model):
```

Comment:
why pass in the user_model?

Reply:
For log-in ill try running it without it

Reply:
Does not work without the fixture